Sources/Nimble/Matchers/BeAKindOf.swift

```diff
@@ -1,14 +1,22 @@
 import Foundation
 
-#if _runtime(_ObjC)
```

Comment:
@ikesyo - I've pushed another update. I'm sorry if my process seems a little cavalier -- I don't know of any way for me to easily test these changes on Linux locally. Do you have any suggestions, or is the process we're going through right now acceptable?